semanticcpg/src/main/scala/io/shiftleft/semanticcpg/utils/ExternalCommand.scala

```diff
@@ -1,10 +1,11 @@
-package io.joern.x2cpg.utils
+package io.shiftleft.semanticcpg.utils
```

Comment:
In order to not make this a breaking change, make `object ExternalCommand` a trait or class and with e.g. an `Impl` suffix and have two `object`s extend it, one in the old and one in the new namespace. Please put a deprecation warning on the old namespace version. That way we dont break anything without a headsup.